### ISSUE TITLE
Exception when running a Next.js project following the quick start and video 

### ISSUE THREAD

**Issue by project707** [prismicio/slice-machine#63]
### Version

- Prismic CLI version (know it by running `prismic --version`): 3.8.3-beta.0
- Node: 15.4.0
-NPM: 7.0.15

### Reproduction

1. Follow the quick start to set up including making a new Next.js project and a new repo: https://www.slicemachine.dev/documentation/next/nextjs-quick-start/
2. Go to http://localhost:3000/test and get the expected "your SliceZone is empty" response, though without the styling shown in the video
3. Following the video create a page with UID test
4. Add a single CTA slice as in the video (but this happens with any slice for me)
5. Refresh http://localhost:3000/test

### What is expected?

I should see a CTA slice rendered in the browser.

### What is actually happening?

An exception is thrown. Here is the CLI output:
`ReferenceError: h is not defined
    at N (webpack-internal:///./node_modules/essential-slices/dist/index.js:1303:3)
    at processChild (/Users/jware/Documents/work/exavault/synced_source/marketing-website/node_modules/react-dom/cjs/react-dom-server.node.development.js:3353:14)
    at resolve (/Users/jware/Documents/work/exavault/synced_source/marketing-website/node_modules/react-dom/cjs/react-dom-server.node.development.js:3270:5)
    at ReactDOMServerRenderer.render (/Users/jware/Documents/work/exavault/synced_source/marketing-website/node_modules/react-dom/cjs/react-dom-server.node.development.js:3753:22)
    at ReactDOMServerRenderer.read (/Users/jware/Documents/work/exavault/synced_source/marketing-website/node_modules/react-dom/cjs/react-dom-server.node.development.js:3690:29)
    at renderToString (/Users/jware/Documents/work/exavault/synced_source/marketing-website/node_modules/react-dom/cjs/react-dom-server.node.development.js:4298:27)
    at Object.renderPage (/Users/jware/Documents/work/exavault/synced_source/marketing-website/node_modules/next/dist/next-server/server/render.js:53:851)
    at Function.getInitialProps (webpack-internal:///./node_modules/next/dist/pages/_document.js:135:19)
    at Function.getInitialProps (webpack-internal:///./pages/_document.js:24:85)
    at loadGetInitialProps (/Users/jware/Documents/work/exavault/synced_source/marketing-website/node_modules/next/dist/next-server/lib/utils.js:5:101)`

And the browser:
![Screen Shot 2020-12-10 at 2 02 43 PM](https://user-images.githubusercontent.com/872643/101835309-6ff7b980-3af0-11eb-927a-8969462e06eb.png)

### Any additional information

I've tried uninstalling and reinstalling prismic-cli, starting projects and Prismic repositories from scratch, upgrading Node, and using different components.
If I run through the same process using Nuxt, everything works fine.

**Comment by davidparys** [comment]
Hey,

Had a similar issue recently, what is the content of you `next.config.js` file?

**Comment by project707** [comment]
FWIW I just tested by creating my Next project manually on v9 to see if it was because Next.js 10 (the default for create-next-app now) is new-ish and maybe not tested with this, but ran into the same issue with default slices. 

The contents of that file:
```
const withTM = require('next-transpile-modules')(['next-slicezone', 'essential-slices']);

module.exports = withTM();
```

**Comment by adrianbarbic** [comment]
I'm encountering the same issue `ReferenceError: h is not defined`, I previously have had this working but is not working today when setting up a new project.

**Comment by davidparys** [comment]
> FWIW I just tested by creating my Next project manually on v9 to see if it was because Next.js 10 (the default for create-next-app now) is new-ish and maybe not tested with this, but ran into the same issue with default slices.
> 
> The contents of that file:
> 
> ```
> const withTM = require('next-transpile-modules')(['next-slicezone', 'essential-slices']);
> 
> module.exports = withTM();
> ```

There seems to be an issue with the `essential-slice` package, as I don't really need the example slices (as they are using theme-ui) I went along and removed it from the webpack from`next.config.js` and it fixed the issue for me.


```Javascript
// next.config.js content
const withTM = require('next-transpile-modules')(['next-slicezone']);

module.exports = withTM({
  target: 'serverless',
  webpack: (config, { isServer }) => {
    // Fixes npm packages that depend on `fs` module
    if (!isServer) {
      config.node = {
        fs: 'empty',
      };
    }
    return config;
  },
});
```

**Comment by hypervillain** [comment]
Thanks for reporting this, can you give a try to essential-slices@1.0.3-alpha.0?

**Comment by samlfair** [comment]
@hypervillain That worked for me.

**Comment by hypervillain** [comment]
Released as 1.0.3. Depending on when you ran `prismic sm --setup`,
you might have to install `theme-ui` as a dependency. Thanks for your patience
